bitnami/fluentd/templates/forwarder-configmap.yaml

```diff
@@ -101,10 +105,12 @@ data:
         flush_interval 5s
       </buffer>
     </match>
-    {{- else }}
+    {{ else }}
```

Comment:
Why removing the "`-`" from "`{{- else }}`"? That will cause the rendered contents to look a bit weird.

Reply:
With the `-` the yaml will remove the final `/n` which makes `extraConfig` to start in the same line as `</match>` and causes fluend to throw an error due to unfinished configuration.